src/index.js

```diff
@@ -24,6 +24,8 @@ const fetchTopStoriesDetails = stories => {
     );
 };
 
+const sortByTime = (a, b) => b.time - a.time;
```

Comment:
@erickbelfy I try to avoid single letter parameters just because they're not obvious. If you can change `a` and `b` to something a little bit more meaningful that would be awesome.